### ISSUE TITLE
The horizontal alignment of an image in IASKCellImage seems to be 1 px out of the center

### ISSUE THREAD

**Issue by funnel20** [futuretap/InAppSettingsKit#402]
When we add a 29x29 @1x image to IASKCellImage to mimic the iOS Settings app, it seems the horizontal alignment of the image is not the same:

Top: InAppSettingsKit
Bottom: iOS Settings app

<img width="581" alt="Screenshot 2019-05-01 at 12 55 12" src="https://user-images.githubusercontent.com/2139844/57014486-a67d0b00-6c10-11e9-977e-07af00aecf4e.png">

Both screenshots are taken on iPhone 7 with iOS 12.2 and InAppSettingsKit 10.2.

**Comment by futuretap** [comment]
Which specifier type are you using in the plist?

**Comment by funnel20** [comment]
We are using `Child Pane` and `IASKButtonSpecifier`.

**Comment by futuretap** [comment]
These specifiers use standard UITableViewCell's. So the layout of the subviews is not in any way altered by IASK. Also, IASK doesn't alter the system default `layoutMargin`/`separatorInset` which is 20 pt for plus-sized phones (including XS max and XR) and 16 pt otherwise.

Settings.app seems to override the 16 pt margin to 15 pt but lets the 20 pt margin for the larger phones untouched. 

Now most apps use standard margins for the rest of their UI and include IASK for only the settings views. It would be strange to change the margins for those apps just for the IASK views.

So in summary, I tend to keep the current behavior because it better matches the overall app UI.

Haven't tried but you can override the `separatorInset` of `IASPAppSettingsViewController.tableView` to achieve a custom inset.

**Comment by funnel20** [comment]
Thanks for explaining this.
I just reported it as a deviation, because I'm not aware of the level of detail that you have to mimic the iOS Settings app.
